content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -396,21 +396,27 @@ direction. By setting the value to `Min` which would select the policy which all
 smallest change in the replica count. Setting the value to `Disabled` completely disables
 scaling in that direction.
 
-### Stabilization Window
+### Stabilization window
+
+The stabilization window is used to restrict the [flapping](#flapping) of
+replicas count when the metrics used for scaling keep fluctuating. The autoscaling algorithm
+uses this window to infer a previous desired state and avoid unwanted changes to workload
+scale.
 
-The stabilization window is used to restrict the flapping of replicas when the metrics
-used for scaling keep fluctuating. The stabilization window is used by the autoscaling
-algorithm to consider the computed desired state from the past to prevent scaling. In
-the following example the stabilization window is specified for `scaleDown`.
+For example, in the following example snippet, a stabilization window is specified for `scaleDown`.
 
 ```yaml
-scaleDown:
-  stabilizationWindowSeconds: 300
+behavior:
+  scaleDown:
+    stabilizationWindowSeconds: 300
 ```
 
 When the metrics indicate that the target should be scaled down the algorithm looks
-into previously computed desired states and uses the highest value from the specified
-interval. In above example all desired states from the past 5 minutes will be considered.
+into previously computed desired states, and uses the highest value from the specified
+interval. In above example, all desired states from the past 5 minutes will be considered.
```

Comment:
```suggestion
interval. In the above example, all desired states from the past 5 minutes will be considered.
```

Reply:
I think this was wrong in the original, FWIW.